docker/docker-build-ee.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Adapted from Geoff Boeing utility script for building a single platform Docker image
+#https://github.com/gboeing/osmnx/blob/2631b95a28300ccffce0c7c6838d269ce708ec1a/environments/docker/docker-build-single_platform.sh
+DOCKERUSER=ryntrnr
```

Comment:
we don't want the dockeruser to be ryntrnr in what we merge here; it should be the orgnisation name.  i.e. we'll build and push an image for the organisation, not as you (or me).